exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -189,6 +189,15 @@ drill.exec: {
   debug: {
     return_error_for_failure_in_cancelled_fragments: false
   }
+  udf: {
+    retry.times: 5,
+    directory: {
+      base: "/tmp",
+      staging: ${drill.exec.zk.root}"/udf/staging",
```

Comment:
zk.root is the root zNode in ZK; it is not a file system node in DFS.

See above comment about defining a new drill.dfs.root property.

I like how you've used the ${} substitution here.